src/npyffi/array.rs

```diff
@@ -12,7 +13,7 @@ const CAPSULE_NAME: &str = "_ARRAY_API";
 /// A global variable which stores a ['capsule'](https://docs.python.org/3/c-api/capsule.html)
 /// pointer to [Numpy Array API](https://numpy.org/doc/stable/reference/c-api/array.html).
 ///
-/// You can acceess raw c APIs via this variable and its Deref implementation.
+/// You can acceess raw C APIs via this variable.
```

Comment:
`acceess` => `access`?

Reply:
Indeed, but I would rather fix this on `main` instead of the backport.

Reply:
See 7c63a12e3ca93995392e6062fbf0a4c5e0fc18c4